lib/glific_web/flows/flow_editor_controller.ex

```diff
@@ -361,7 +362,13 @@ defmodule GlificWeb.Flows.FlowEditorController do
         [%{id: "#{c.id}", name: c.name, type: "contact", extra: c.id} | acc]
       end)
 
-    json(conn, %{results: recipients})
+    groups =
```

Comment:
@AkhileshNegi  why are we doing string interpolation here? Isn't that already a string?

Reply:
and there is a better way to write this

groups =
  for g <- Groups.list_groups(%{}), into: [], do:
    %{id: "#{g.id}", name: g.label, type: "group", extra: g.id}

Reply:
@siddhant3030 Where exactly are you referring the change or string interpolation?
Only difference I see is use of for loop instead of Enum. 
Rest map looks same to me
Also the PR is not completed yet and not assigned for review. I'll assign you for review once done